### DISCUSSION TITLE
Case insensitive authors/names/albums

### DISCUSSION THREAD

**Discussion by RobertSpir** [gerbera/gerbera#3147] [Q&A] [answered]
It is possible to set the indexing to case insensitive, so that names like Author1 and author1 will be treated as the same author?

**Comment by KarlStraussberger** [comment, marked as answer]
Yes, there is an option `case-sensitive-tags` in https://docs.gerbera.io/en/latest/config-import.html

This is only in our development version which will be released within the next days.
